rlclientlib/schema/v2/FileFormat.fbs

```diff
@@ -0,0 +1,25 @@
+include "Event.fbs";
+
+namespace reinforcement_learning.messages.flatbuff.v2;
+
+table JoinedEvent {
+    event: [ubyte];
+    timestamp: TimeStamp;
+}
+
+table JoinedPayload {
+    events: [JoinedEvent];
+}
+
+table KeyValue {
+    key: string;
+    value: string;
+}
+
+table FileHeader {
+    join_time: TimeStamp;
+    properties: [KeyValue];
+}
+
+root_type JoinedPayload;
+root_type FileHeader;
```

Comment:
just fyi it seems that in c++ only one file is generated for this schema named: `FileFormat_generated.h` from that file only `GetFileHeader` is available, so if these stay in the same file we will have to do `flatbuffers::GetRoot<reinforcement_learning::messages::flatbuff::v2::JoinedPayload>(buffer)` to get JoinedPayload root